frontend/test/unit/specs/plugins/errors.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { Context } from "@nuxt/types"
```

Comment:
I did start with that, but the function the plugin puts into `$processFetchingError` still requires `$sentry` and `useAnalytics`. It also would require further exporting things from the plugin module that aren't really needed anywhere other to make them testable, and would be testing such small units that it would kind of miss the bigger picture (is how I felt).

Happy to change this if you see a way to make the tests much simpler, and not require mocking `$sentry` and `useAnalytics` still, but if they still require that, then the only change is one that also additionally tests that the plugin is configured correctly (calls `inject` with the right parameters).

Reply:
Not directly related to this comment, but after spending some time this morning investigating the problems with "search time" event, I realized that this change would also require the changes that would make `useAnalytics` usable on the server (see #3775)